sdk/nodejs/helm/v2/helm.ts

```diff
@@ -367,10 +361,9 @@ export function fetch(chart: string, opts?: ResolvedFetchOpts) {
         if (opts.repo !== undefined)        { flags.push(`--repo ${path.quotePath(opts.repo)}`);               }
         if (opts.untardir !== undefined)    { flags.push(`--untardir ${path.quotePath(opts.untardir)}`);       }
         if (opts.username !== undefined)    { flags.push(`--username ${shell.quote([opts.username])}`);  }
-        if (opts.home !== undefined)        { flags.push(`--home ${path.quotePath(opts.home)}`);               }
         if (opts.devel === true)            { flags.push(`--devel`);                                           }
         if (opts.prov === true)             { flags.push(`--prov`);                                            }
         if (opts.verify === true)           { flags.push(`--verify`);                                          }
     }
-    execSync(`helm fetch ${shell.quote([chart])} ${flags.join(" ")}`);
+    execSync(`helm fetch ${shell.quote([chart])} ${flags.join(" ")}`, { env });
```

Comment:
Are we sure we want to pass all env variables as opposed to passing `HELM_HOME` specifically? Is there a combination of env variables that would change the behavior of helm between two Pulumi versions, and therefore break somebody?

Reply:
I suppose it's possible. The advantage of passing everything is that it makes it easier for users to configure Helm behavior without requiring us to change code, but maybe it would be better to limit the change here.

Reply:
The default for `execSync` was to pass `process.env` down to the subprocess.  So I think we should keep that behaviour by default (and add `HELM_HOME`).

Reply:
Indeed! The same applies to .NET, actually, if I read the docs correctly. Therefore, we can simplify the PR and not retrieve/pass env variables, only add `HELM_HOME` if needed.

Reply:
@lukehoban [This comment](https://stackoverflow.com/questions/7824789/nodejs-set-environment-variable-for-exec#comment57015051_7825732) indicates that setting `env` only sets the passed values, which is why I copied `process.env` explicitly.

Reply:
@lblackstone That makes sense to me. I think we don't need to retrieve/pass them for .NET - just add the extra one.